tox.ini

```diff
@@ -19,6 +19,7 @@ setenv =
     BUILD=linter
 commands =
     flake8 sphinxcontrib setup.py
+    isort --check --diff .
```

Comment:
Does this modify the files, or just check them? It would be useful to have a tox target to run the tool and modify the files in the current working directory, to make them comply with the linter without having to manually make those changes (for contributors who do not have isort configured in their editor).

Reply:
Good idea.

This command, as is, only checks the file.

I added a new tox environment `style` that contributors can use to run the isort without the need to install isort themselves (let me know if you prefer a different name for this).

Unfortunately, this command can't also be used for the `linter` environment as without `--check` isort won't return a non-zero status code and with `--check` it won't write out the files. So I think we're stuck with two tox environments.

I added some docs to `developers.rst` that instructs how to use this command. Feedback is welcome.

Reply:
That's exactly what I had in mind. Thanks!